test/ng/directive/formSpec.js

```diff
@@ -386,6 +386,9 @@ describe('form', function() {
       doc = jqLite('<form ng-submit="submitMe()">' +
                      '<input type="submit" value="submit">' +
                    '</form>');
+      // Support: Chrome 60+ (on Windows)
+      // We need to add the form to the DOM in order for `submit` events to be properly fired.
+      window.document.body.appendChild(doc[0]);
```

Comment:
We [empty body][1] before each test.

[1]: https://github.com/angular/angular.js/blob/3650723501ca5f50003192bc4f94c77cef711e1a/test/helpers/testabilityPatch.js#L23